core/src/main/scala/com/salesforce/op/filters/FeatureDistribution.scala

```diff
@@ -63,6 +63,8 @@ case class FeatureDistribution
   nulls: Long,
   distribution: Array[Double],
   summaryInfo: Array[Double],
+  moments: Option[Moments] = None,
+  cardEstimate: Option[TextStats] = None,
```

Comment:
Are these optional for backwards compatibility of model insights?

Reply:
correct!